static/js/main.js

```diff
@@ -53,6 +53,13 @@ require.config({
 		'sjcl': 'libs/sjcl',
 		'bootstrap-file-input': 'libs/bootstrap.file-input',
 		'webfont': 'libs/webfont',
+		'axolotl': 'libs/axolotl/libaxolotl-javascript-1.3.0',
+		'protobufjs': 'libs/axolotl/protobufjs.min',
+		'traceur-runtime': 'libs/axolotl/traceur-runtime.min',
+		'axolotl-crypto': 'libs/axolotl/axolotl-crypto',
+		'axolotlCryptoCurve25519': 'libs/axolotl/axolotl-crypto-curve25519',
+		'ByteBuffer': 'libs/axolotl/ByteBufferAB.min',
+		'Long': 'libs/axolotl/Long.min',
```

Comment:
`libaxolotl-javascript` has support for `require.js`, but defining the dependencies explicitly here won't hurt...